README.md

```diff
@@ -2,10 +2,62 @@
+
+### Run the project
+Finally, to run this project locally:
+```
+npm run dev
+```
+
+## Pulling data from [Zoning API](https://github.com/NYCPlanning/ae-zoning-api)
+This app relies heavily on data from the Zoning API backend application. It makes use of some front end libraries to make pulling data and managing the state of that data easier:
+
+* [Axios](https://axios-http.com/docs/intro) - A framework-agnostic HTTP client for javascript and typescript applications
+* [TanStack Query](https://tanstack.com/query/v4) - This is a general purpose asynchronous state management library for React. You can read more about what problems TanStack Query strives to solve [here](https://tanstack.com/query/v4/docs/react/overview#motivation). It provides useful state management such as caching and managing loading states.
+* [Kubb](https://www.kubb.dev/introduction) - A code generation library that reads [OpenAPI specification](https://www.openapis.org/) documents and generates typescript "client" libraries (sometimes also referred to as "SDKs"). It features a set of plugins to generate code for various JS/TS libraries, including Axios and TanStack Query. By pointing Kubb at the [OpenAPI document for Zoning API](https://github.com/NYCPlanning/ae-zoning-api/blob/main/openapi/openapi.yaml), we can avoid writing a lot of boilerplate code.
+
+### Regenerating the Zoning API client code
+When the OpenAPI spec file for Zoning API changes, we will need to regenerate the client code in this repo. To do that, simply run `npm run generate`. Kubb is configured to put all generated code in `/src/gen`. Note that even though that code is generated, we do want to commit it and includes updates in PRs as necessary. A developer should **never** make manually changes or additions to code in the `/src/gen` folder. It should be purely what is outputted by Kubb.
+
+> You may see an error in your terminal after running `npm run generate` referring to the "path" argument being undefined. The root cause of this is still being looked into but it doesn't seem to affect the generated code so you can safely ignore it for now.
+
+Unless you think the contents of the Zoning API OpenAPI spec file have changed since the last time the client code was generated, you can safely skip this step.
+
+### Using the Zoning API client code
+Once you have this application configured to point to a running instance of Zoning API, you can start to pull in data. To query data from the Zoning API in our React code, we only need to import and call the hooks generated by Kubb. The `Providers` component makes sure our entire application is wrapped by TanStack Query's `QueryClientProvider` component, so we should be able to call the hooks within any component. To query data, simply import the hook corresponding to the data you need from the `/src/gen` folder and call it within your React component.
```

Comment:
I think the part about us using TanStack Query is no longer true for this repo. I think it's better to just remove this part all together or get others' input to document how we do data fetching here.

Reply:
Sounds good. We can get @TangoYankee to take a look at his leisure.

Reply:
> I think the part about us using TanStack Query is no longer true for this repo. I think it's better to just remove this part all together or get others' input to document how we do data fetching here.


Correct. We use axios to make requests via the loader functions because the loaders execute on the server side. We do still use kubb generated functions to configure the axios calls. An example is: [`findCommunityDistrictsByBoroughId`](https://github.com/NYCPlanning/ae-cp-map/blob/e877149a449da8e51ecbbf45dac67f015210978d/app/root.tsx#L78)